schema/ListedLicense.xsd

```diff
@@ -29,46 +53,214 @@
+					<xhtml:p>
+						<xhtml:code>name</xhtml:code> is the name of the license, as provided by the license steward.  In the absence of an external license steward, the name is chosen by the SPDX legal team.
+					</xhtml:p>
+				</documentation>
+			</annotation>
+		</attribute>
+		<attribute name="listVersionAdded" type="string">
+			<annotation>
+				<documentation xml:lang="en">
+					<xhtml:p>
+						<xhtml:code>listVersionAdded</xhtml:code> is the version of the license list which first included the license.  This impacts the portability of a license identifier, because identifiers introduced in a later version may not be supported by tooling which targets an earlier version.
+					</xhtml:p>
+				</documentation>
+			</annotation>
+		</attribute>
+		<attribute name="deprecatedVersion" type="string">
+			<annotation>
+				<documentation xml:lang="en">
+					<xhtml:p>
+						<xhtml:code>deprecatedVersion</xhtml:code> is the version of the license list which first deprecated the license.  SPDX authors are encouraged to upgrade to the most recent equivalent identifier supported by their tooling.  Future major releases of the license list may remove deprecated entries completely, after which new tools may no longer be able to understand them.
+					</xhtml:p>
+				</documentation>
+			</annotation>
+		</attribute>
 	</complexType>
 	<complexType name="crossRefsType">
+		<annotation>
+			<documentation xml:lang="en">
+				<xhtml:p>
+					<xhtml:code>crossRefsType</xhtml:code> is a set of URIs for the license or exception.  When the license steward has a URI devoted to the license, it must be the first entry in <xhtml:code>crossRefsType</xhtml:code>.  The order in which the other child elements are listed is not significant.
+				</xhtml:p>
+			</documentation>
+		</annotation>
 		<sequence>
-			<element name="crossRef" type="string" minOccurs="1" maxOccurs="unbounded"/>
+			<element name="crossRef" type="string" minOccurs="1" maxOccurs="unbounded">
+				<annotation>
+					<documentation xml:lang="en">
+						<xhtml:p>
+							<xhtml:code>crossRef</xhtml:code> is a URI related to the license or exception.  Two main types are supported: resources devoted to the license itself (either by the license steward or third parties) and examples of use in prominent projects.  Dead links may be supplemented by archived replacements (e.g. through the <xhtml:a href="https://archive.org/">Internet Archive</xhtml:a>).
+						</xhtml:p>
+					</documentation>
+				</annotation>
+			</element>
 		</sequence>
 	</complexType>
 	<complexType name="altType" mixed="true">
-		<group ref="tns:formattedFixedTextGroup" minOccurs="0" maxOccurs="unbounded"/>
-		<attribute name="name" type="string" use="required" />
-		<attribute name="match" type="string" use="required" />
+		<annotation>
+			<documentation xml:lang="en">
+				<xhtml:p>
+					<xhtml:code>altType</xhtml:code> represents alternative content, where the SPDX considers several alternatives legally equivalent.
```

Comment:
This may be OK, but in practice we have a lot of `.+` matches which really could include text not legally equivalent.  Should we include something with a little more "wiggle room" in the language?  For example - "represents alternative content commonly found in the license text, where the SPDX considers typical usage of the alternative text legally equivalent."

@jlovejoy thoughts?

Reply:
> This may be OK, but in practice we have a lot of `.+` matches which really could include text not legally equivalent.

I've added a warning about this in 7d44198 → deec602.